src/auction.vy

```diff
@@ -22,6 +40,59 @@ event OwnershipTransferred:
     new_owner: indexed(address)
 
 
+
+event AuctioneerChanged:
+    previous_auctioneer: indexed(address)
+    new_auctioneer: indexed(address)
+
+
+
+@external
+def change_auction_duration(new_duration: uint256):
+    """
+    @dev Changes the duration of the auction.
+    @param new_duration The new duration of the auction.
+    """
+    self._check_owner()
+    self.auction_duration = new_duration
+
+@external
+def change_sniping_timer(new_timer: uint256):
+    """
+    @dev Changes the time added to an auction if a bid is made near its end.
+    @notice Can only be called by the contract owner.
+    @param new_timer The new extension time in seconds.
+    """
+    self._check_owner()
+    assert new_timer > 0 and new_timer <= 86400, "Invalid timer: must be between 1 second and 24 hours"
+    self.extension_time_seconds = new_timer
+
+@external
+def change_minimum_bid_increment_percentage(new_percentage: uint256):
+    self._check_owner()
+    self.minimum_bid_increment_percentage = new_percentage
+
+@external
+def change_extension_time_seconds(new_extension_time: uint256):
+    self._check_owner()
+    self.extension_time_seconds = new_extension_time
```

Comment:
Why there are 2 functions for the same thing? `change_extension_time_seconds ` and `change_sniping_timer`

Reply:
https://github.com/BenderBendingRodriguez4/auction_house_contracts/pull/1/commits/a1e26e61d73650e4d9365327ca4b92425a2b8778 

Fixed

Reply:
Fixed here: https://github.com/BenderBendingRodriguez4/auction_house_contracts/pull/1/commits/a1e26e61d73650e4d9365327ca4b92425a2b8778